create-k8s-chained-sessions.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+# global variables
+declare K8S_PROFILE_ID
+declare CHAINED_PROFILE_IDS="name,id\n"
+
+###### FUNCTIONS ######
+#
+# function to create a chained leapp session given a parent session id
+# Args:
+# 1: parent session id name from leapp
+# appends new session id from the new chained session to CHAINED_PROFILE_IDS
+function createLeappSession() {
+    parent_session_name=$1
+    chained_session_name="chained-from-${parent_session_name}"
+    echo "starting session for ${parent_session_name} to get role arn"
+    # this has funky piping because `--filter` is a fuzzy lookup and `panorama-k8s-playground` fuzzy matches `panorama-k8s-playground-2` as the first result
+    parent_session_id=$(leapp session list -x --filter="Session Name=${parent_session_name}" --no-header | sort -k1 -r | sed -n 1p | awk '{print $1}')
+    # start leapp session
+    leapp session start --sessionId $parent_session_id
+    # call to aws to get the role arn for `TerraformRole`
+    role_arn=$(aws iam get-role --role-name TerraformRole --query Role.Arn | tr -d '"')
+    # stop the leapp session
+    leapp session stop --sessionId $parent_session_id
+
+    echo "creating new session"
+    # create new chained leapp session from parent
+    leapp session add --providerType aws --sessionType awsIamRoleChained \
+        --sessionName $chained_session_name --region us-east-1 \
+        --roleArn $role_arn --parentSessionId $parent_session_id \
+        --profileId $K8S_PROFILE_ID
+    # add session id from the new session to CHAINED_PROFILE_IDS
+    chained_session_id=$(leapp session list --columns=ID --filter="Session Name=${chained_session_name}" --no-header)
```

Comment:
Good question! Technically it would if you re-ran it, but since `chained-from-panorama-k8s-playground` gets created first in the loop it doesn't have any other fuzzy results to conflict with.  Since this is only meant to be run once as an initial setup and would fail in many ways if you repeated it without resetting I don't think that's a crucial work around to add here. What are your thoughts?